src/scheme.rs

```diff
@@ -71,6 +71,8 @@ pub enum Scheme {
     S3,
     /// [oss][crate::services::oss]: Aliyun Object Storage Services
     Oss,
+    /// [webhdfs][crate::services::webhdfs]: WebHDFS RESTful API Services
+    WebHdfs,
```

Comment:
Please make sure all scheme in alphabet order.

Reply:
Do you mean `WebHdfs` -> `Webhdfs`?